msgcntr/messageforums-app/src/webapp/js/forum.js

```diff
@@ -374,6 +374,10 @@ function doAjax(messageId, topicId, self){
                     if ($(self).parent('td').size() === 1) {
                         var thisTheadClassArr = $(thisRow).prop('class').split(' ');
                         var thisThread = thisTheadClassArr[thisTheadClassArr.length - 1];
+                        var unread = parseInt($('.hierItemBlock.' + thisThread + ' .childrenNewNumber').text());
+                        if(unread > 0) {
```

Comment:
Small thing. Can you add a space after the if? It makes the code a bit clearer.

Reply:
Done!